README.md

```diff
@@ -1,157 +1,163 @@
+| [Rack](#rack)                   | rack-tracer                            | >= 0.1             |
+| [Rails](#rails)                 | rails-instrumentation                  | >= 3.0.0           |
+| [Redis](#redis)                 | redis-instrumentation                  | >= 4.0.0           |
+| [RestClient](#restclient)       | restclient-instrumentation             | >= 1.5.0           |
+| [Sequel](#sequel)               | sequel-instrumentation                 | >= 3.47.0          |
+| [Sidekiq](#sidekiq)             | sfx-sidekiq-opentracing                | >= 0.7.0           |
+| [Sinatra](#sinatra)             | sinatra-instrumentation                | >= 1.0.0           |
+
+Instrumentation for routes using the following web servers is provided through
+Rack. If you use a framework that builds on top of Rack, such as Rails or
+Sinatra, install the `rack-tracer` instrumentation with your dependency manager
+or with the bootstrap utility. In these cases, the routes through the web
+server are automatically traced.
+
+## Install the SignalFx Tracing Library for Ruby
+
+Follow these steps to install the tracing library. You can either use the
+bootstrap utility to install the tracing library and its dependencies or
+manually install everything. 
+
+The bootstrap utility updates your Gemfile and installs the required
+dependencies for you. For information about the bootstrap utility, see the
+[sfx-rb-trace-bootstrap](bin/sfx-rb-trace-bootstrap) file.
+
+The steps assume you have RubyGems and Bundler.
+
+### Install the library with the bootstrap utility
+
+1. Install the tracing library:
+   ```bash
+   $ gem install signalfx-tracing
+   ```
+2. View the list of instrumentations you can install with the bootstrap utility:
+   ```bash
+   $ sfx-rb-trace-bootstrap --list
+   ```
+3. Use the bootstrap utility to install applicable instrumentations for your
+   application. For example, this is how you add Rails and Redis:
+   ```bash
+   $ sfx-rb-trace-bootstrap --install-deps rails-instrumentation,redis-instrumentation
+   ```
+   For information about instrumentation names, see
+   [Supported libraries](#supported-libraries). If you configure Rails
+   instrumentation, it also configures Active Record instrumentation, so you don't
+   need to instrument both.
+
+### Manually install the library
+
+1. Download the [latest release](https://github.com/signalfx/signalfx-ruby-tracing/releases/latest) of the tracing library.
+2. Add `signalfx-tracing` to your application's Gemfile:
+   ```bash
+   $ gem 'signalfx-tracing'
+   ```
+3. Add each applicable instrumentation to your application's Gemfile. For
+   example, this is how you add Rails and Redis:
+   ```bash
+   $ gem 'rails-instrumentation'
+   $ gem 'redis-instrumentation'
+   ```
+   For information about instrumentation names, see
+   [Supported libraries](#supported-libraries). If you configure Rails
```

Comment:
We no longer document installing from a cloned repo so this should probably be "see supported libraries and their current instrumentation versions in `gem.deps.rb`."